### PR TITLE
Fix flow static typing for src/utils.

### PR DESCRIPTION
**Pull request by TomaszKolek**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@TomaszKolek, it looks like you've already signed the Dropbox CLA. Thanks!

File: src/account-info/AccountDetailsScreen.js [hunk 1]
```diff
@@ -2,18 +2,14 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
+import { NOT_FOUND_USER } from '../constants';
```
Comment (TomaszKolek) [inline src/account-info/AccountDetailsScreen.js]: Check this out, I created a file when I store common `NOT_FOUND_USER` object

File: src/api/apiFetch.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ export const apiCall = async (
     const response = await timeout(
       await apiFetch(auth, route, params),
       () => { throw new Error(`Request timed out @ ${route}`); },
-      shouldTimeout,
+      // shouldTimeout, # WTF ITS BOOL ;-( WE NEED NUMBER
```
Comment (TomaszKolek) [inline src/api/apiFetch.js]: Thats strange, I commented it out now - it should be a number, boolean has no sense here.

File: src/message/Timestamp.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ export default class Timestamp extends React.PureComponent {
     const { timestamp, twentyFourHourTime } = this.props;
     return (
       <Text style={styles.time}>
-        {shortTime(timestamp * 1000, twentyFourHourTime)}
+        {shortTime(new Date(timestamp * 1000), twentyFourHourTime)}
```
Comment (TomaszKolek) [inline src/message/Timestamp.js]: `shortTime` takes `Date` object - it was typed by an author.

File: src/utils/notifications.android.js [hunk 4]
```diff
@@ -1,4 +1,5 @@
 /* @flow */
+/* @flow */
```
Comment (TomaszKolek) [inline src/utils/notifications.android.js]: oups

File: src/api/apiFetch.js (hunk 2 above)
Comment (borisyankov) [inline src/api/apiFetch.js]: Flow catches errors I guess :)

**Comment by borisyankov** [comment]
Try enabling "flowtype/no-weak-types" in eslintrc and fixing some of the related issues.

**Comment by borisyankov** [comment]
Please, rebase and fix so I can merge.

**Pull request merged**